junit-5/src/test/java/runners/BstackRunner.java

```diff
@@ -0,0 +1,126 @@
+package runners;
+
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.junit.jupiter.api.extension.*;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import utils.SetupLocalTesting;
+
+import java.io.FileReader;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.*;
+import java.util.stream.Stream;
+
+public class BstackRunner implements TestTemplateInvocationContextProvider {
+    JSONObject mainConfig, browserConfig, profileConfig, testConfig, platformConfig, commonCapsConfig;
+    HashMap<String, String> allCapsMap,commonCapsMap;
+    WebDriver driver;
+    DesiredCapabilities capabilities;
+    String username, accesskey, server;
+
+    public String[] setupCredentials(){
+        try{
+            JSONParser parse = new JSONParser();
+            mainConfig = (JSONObject) parse.parse(new FileReader("src/test/resources/caps.json"));
+            server = (String) mainConfig.get("server");
+            username = System.getenv("BROWSERSTACK_USERNAME");
+            if(username == null){
+                username = (String) mainConfig.get("username");
+            }
+            accesskey = System.getenv("BROWSERSTACK_ACCESS_KEY");
+            if(accesskey == null){
+                accesskey = (String) mainConfig.get("accesskey");
```

Comment:
`accessKey = (String) config.get("key");` this was used for sel4, can we keep all the variable names and keys same?

Reply:
have modified the variable names to "key" and "user" as per the capabilities doc for [Selenium 3](https://www.browserstack.com/automate/capabilities?tag=selenium-2-3)